examples/simple-page/index.js

```diff
@@ -0,0 +1,16 @@
+var bemxjst = require('bem-xjst');
+var data = require('./data');
+
+// Convert JSON from backend to BEMJSON
```

Comment:
I think it'd be better to provide comment not for each section but for each line of code (otherwise it may be misunderstood: why this `require` converts data?)